test/manager.js

```diff
@@ -51,5 +52,27 @@ var manager = module.exports = {
 				next();
 			}).catch(next);
 		}, cb);
+	},
+
+	createUser : function(username, email, password, cb){
+		// Create new user to test add project and login functionality
+		var rounds = 1;
+		bcrypt.genSalt(rounds, function(err, salt) {
+			bcrypt.hash(password, salt, null/* progress callback */, function(err, hash) {
+				// store new user in db
+				db('users')
+					.insert({
+						username: username,
+						email: email,
+						password: hash
+					})
+					.then(function () {
+						cb
+					})
+					.catch(function(error){
+						console.log(error);
+					});
```

Comment:
Should do:
```js
.catch(cb);
```

So that if there is a database error, the test will fail instead of timeout.